### PR TITLE
Perferences modal is now pretty

### PR DESCRIPTION
**Pull request by SaraVieira**
Fixes CSB-23

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3715/commits/2ef8cf2655a8229b3565812dbbd8c53e45e51b73 failed.

**Comment by DannyRuchtie** [comment]
Sara starting to look good! 
Could we have the same amount of padding in the menu list. 


Screenshot 2020-04-15 at 14 03 16

**Comment by DannyRuchtie** [comment]
Screenshot 2020-04-15 at 14 08 37
Screenshot 2020-04-15 at 14 08 17

Not sure why but the border is loading blue in a lot of the other themes.

**Comment by DannyRuchtie** [comment]
Screenshot 2020-04-15 at 14 09 09

Theme related: The contrast in Night owl in good. 
The foreground color should be dark?

File: packages/app/src/app/pages/common/Modals/PreferencesModal/PaymentInfo/index.tsx [hunk 1]
```diff
@@ -33,32 +61,75 @@ export const PaymentInfo: FunctionComponent = () => {
...
+            })}
+          >
+            {Icon(brand)()}
+          </Element>
+          <Element>
+            <Element>
+              <Text weight="bold" size={3}>
+                {brand}
+              </Text>{' '}
+              <Text variant="muted">ending in ****</Text>
```
[the remark this answers is not in the document]
Comment (SaraVieira) [reply packages/app/src/app/pages/common/Modals/PreferencesModal/PaymentInfo/index.tsx:102]: No, thats a design choice ahah

**Comment by DannyRuchtie** [comment]
Screenshot 2020-04-15 at 14 37 02
Screenshot 2020-04-15 at 14 39 02
Color contrast foreground background not implemented?

**Comment by DannyRuchtie** [comment]
Screenshot 2020-04-15 at 18 33 15
@SaraVieira  these buttons probably need to be updated right?

File: package.json [hunk 2]
```diff
@@ -35,7 +35,7 @@
     "build:clean": "lerna run build:clean --scope app --scope homepage && rimraf www",
     "build:common": "lerna run build:dev --scope @codesandbox/common --stream",
     "build:components": "lerna run build:dev --scope @codesandbox/components --stream",
-    "build:deps": "lerna run build:dev",
+    "build:deps": "lerna run build:dev --scope @codesandbox/components --stream && lerna run build:dev",
```
Comment (CompuIves) [inline package.json:38]: Do we still need this? Lerna should be able to find the right order and most efficient parallelization of building based on dependencies.

File: package.json [hunk 3]
```diff
@@ -107,7 +107,7 @@
     ]
   },
   "engines": {
-    "node": "^10.0.0"
+    "node": "^10 || ^12"
```
Comment (CompuIves) [inline package.json:110]: If everything still works with this we can keep it!

File: packages/app/src/app/components/SubscribeForm/CheckoutForm/elements.ts [hunk 4]
```diff
@@ -1,12 +1,8 @@
 import styled from 'styled-components';
```
Comment (CompuIves) [inline packages/app/src/app/components/SubscribeForm/CheckoutForm/elements.ts:1]: Important to check if `/patron` and `/pro` changed styling after this change.

File: packages/app/src/app/components/SubscribeForm/CheckoutForm/index.tsx [hunk 5]
```diff
@@ -4,10 +4,12 @@ import {
   CardElement,
```
Comment (CompuIves) [inline packages/app/src/app/components/SubscribeForm/CheckoutForm/index.tsx:4]: Same here! Important to check `/patron` and `/pro`.

File: packages/app/src/app/pages/common/GithubIntegration/index.tsx [hunk 6]
```diff
@@ -22,7 +22,7 @@ export const GithubIntegration: FunctionComponent<Props> = ({
 
   return (
     <Integration
-      bgColor="#4078c0"
+      bgColor="#0971f1"
```
Comment (CompuIves) [inline packages/app/src/app/pages/common/GithubIntegration/index.tsx:25]: This one is tricky, the old one was based on the brand colors from GitHub. I'm fine with changing this, but we should keep it in mind cc @DannyRuchtie .

File: packages/app/src/app/pages/common/Modals/PreferencesModal/Integrations/index.js [hunk 7]
```diff
@@ -2,14 +2,16 @@ import React from 'react';
...
+import { Text } from '@codesandbox/components';
 
 import { Container } from './elements';
-import { Title } from '../elements';
 
 export function Integrations() {
   return (
     <div>
-      <Title>Integrations</Title>
+      <Text size={4} marginBottom={4} block variant="muted" weight="bold">
```
Comment (CompuIves) [inline packages/app/src/app/pages/common/Modals/PreferencesModal/Integrations/index.js:12]: Why this title change?

File: packages/common/package.json [hunk 8]
```diff
@@ -45,6 +45,7 @@
   "dependencies": {
     "@babel/plugin-transform-flow-strip-types": "^7.0.0",
     "@babel/polyfill": "^7.4.4",
+    "@codesandbox/components": "^0.0.2",
```
Comment (CompuIves) [inline packages/common/package.json:48]: Is this still needed? Not sure if common is still using components.

File: packages/common/src/themes/codesandbox-black.js [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
+import { colors as tokens } from '@codesandbox/components/lib/design-language/colors';
 
 /*
   we use dot to convert objects to vscode dot notation
 
   the object style is better authoring experience, it helps
-  organising the file better, let's us lint the file and find
-  duplicates / classhing styles.
+  organizing the file better, let's us lint the file and find
+  duplicates / clashingstyles.
```
Comment (CompuIves) [inline packages/common/src/themes/codesandbox-black.js:9]:
```suggestion
  duplicates / clashing styles.
```

File: packages/app/src/app/pages/common/GithubIntegration/index.tsx (hunk 6 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/common/GithubIntegration/index.tsx:25]: This was asked by Danny

**Comment by CompuIves** [comment]
image

In CodeSandbox theme this happens on hover, maybe we need to fix this in the theme itself?

File: packages/app/src/app/pages/common/Modals/PreferencesModal/Integrations/index.js (hunk 7 above)
Comment (SaraVieira) [reply packages/app/src/app/pages/common/Modals/PreferencesModal/Integrations/index.js:12]: Sowy, fixed

File: packages/common/package.json (hunk 8 above)
Comment (SaraVieira) [reply packages/common/package.json:48]: It does use it, we needed to keep it for preferences stuff

File: packages/app/src/app/components/SubscribeForm/CheckoutForm/index.tsx (hunk 5 above)
Comment (SaraVieira) [reply packages/app/src/app/components/SubscribeForm/CheckoutForm/index.tsx:4]: Fixed and working

File: packages/app/src/app/components/SubscribeForm/CheckoutForm/elements.ts (hunk 4 above)
Comment (SaraVieira) [reply packages/app/src/app/components/SubscribeForm/CheckoutForm/elements.ts:1]: Fixed and working

**Pull request merged**